test/tangy-form-editor_test.html

```diff
@@ -48,6 +48,33 @@
         </tangy-form-editor>
       </template>
     </test-fixture>
+ 
+    <test-fixture id="PrintFixture">
+      <template>
+        <tangy-form-editor print>
```

Comment:
Instead of having something like a "print" button similar to the "preview" button, entering print mode is a top level API that the outside application will have to apply via the `print` attribute. The reasoning is if we go the "print" button route, the print mode would be only inside of the `<tangy-form-editor>` component and the outside app would still have junk laying around that the user does not want printed out. `<tangy-form-editor>`. `<tangy-form-editor>` could emit an `print-mode-enabled`/`print-mode-disabled` events, but that's a lot of juggling for not much benefit. Let the outside app craft the UX for entering and exiting print mode.